testcases/OpTestDKGSB.py

```diff
@@ -0,0 +1,350 @@
+
+            # After PK reset, zero the HMC secure-boot parameters
+            self.ensure_hmc_available()
+            # Shutdown LPAR and wait ~30s
+            self.shutdown_lpar(host_name, system_name, sleep_time=30)
+            # Configure secure-boot related params to zero
+            config_params = (
+                "keystore_signed_updates_without_verification=0,"
+                "keystore_signed_updates=0,"
+                "linux_dynamic_key_secure_boot=0,"
+                "keystore_kbytes=0"
+            )
+            self.configure_secure_boot(host_name, system_name, config_params)
+            # Allow HMC to settle
+            time.sleep(10)
+            # Get and log current status
+            output = self.get_secure_boot_status(host_name, system_name)
+            log.info("Current secure boot status after resetting to zero:")
+            for line in output:
+                log.info(line)
+            # Start the LPAR
+            self.start_lpar(host_name, system_name)
+
+            # Cleanup via SecvarctlTest.tearDown()
+            try:
+                if repo_root:
+                    sec.home = repo_root
+                sec.tearDown()
+            except Exception as e:
+                log.warning(f"secvarctl tearDown reported an error: {e}")
+
+            log.info("Secure boot reset completed and secvarctl artifacts cleaned up")
+        except Exception as e:
+            error_msg = f"Failed to reset secure boot settings: {str(e)}"
+            log.error(error_msg)
+            self.fail(error_msg)
+
+    def enable_secure_boot(self, host_name, system_name):
+        """
+        Enable secure boot (set to mode 2)
+        """
+        try:
+            log.info(f"Enabling secure boot for host: {host_name}, system: {system_name}")
+            self.shutdown_lpar(host_name, system_name)
+            self.configure_secure_boot(host_name, system_name, "secure_boot=2")
+            self.start_lpar(host_name, system_name)
+            log.info("Secure boot enabled successfully")
+            
+        except Exception as e:
+            error_msg = f"Failed to enable secure boot: {str(e)}"
+            log.error(error_msg)
+            self.fail(error_msg)
+
+    def disable_secure_boot(self, host_name, system_name):
+        """
+        Disable secure boot (set to mode 0)
+        """
+        try:
+            log.info(f"Disabling secure boot for host: {host_name}, system: {system_name}")
+            self.shutdown_lpar(host_name, system_name, sleep_time=30)
+            self.configure_secure_boot(host_name, system_name, "secure_boot=0")
+            time.sleep(5)
+            self.start_lpar(host_name, system_name)
+            log.info("Secure boot disabled successfully")
+        except Exception as e:
+            error_msg = f"Failed to disable secure boot: {str(e)}"
+            log.error(error_msg)
+            self.fail(error_msg)
+
```

Comment:
For enable and disable secure boot why can't we use the functions from static guest secure boot? Have you explored that option?